samples/extensions/full_screen_exclusive/full_screen_exclusive.cpp

```diff
@@ -144,7 +142,7 @@ void FullScreenExclusive::init_instance(const std::vector<const char *> &require
 	std::vector<const char *> active_instance_extensions(required_instance_extensions);
 
 #if defined(VKB_DEBUG) || defined(VKB_VALIDATION_LAYERS)
-	active_instance_extensions.push_back(VK_EXT_DEBUG_UTILS_EXTENSION_NAME);
+	active_instance_extensions.push_back(VK_EXT_DEBUG_REPORT_EXTENSION_NAME);
```

Comment:
Should be `VK_EXT_DEBUG_UTILS_EXTENSION_NAME`. With that, and the other changes this works fine in my office system too.

Reply:
perfect; thanks for checking.